account_fiscal_position_rule_stock/stock.py

```diff
@@ -75,3 +75,109 @@ def _prepare_invoice(self, cr, uid, picking, partner, inv_type,
         result['fiscal_position'] = picking.fiscal_position and \
             picking.fiscal_position.id
         return result
+
+
+class stock_picking_out(osv.Model):
+    _inherit = "stock.picking.out"
+    _columns = {
+        'fiscal_position': fields.many2one(
+            'account.fiscal.position', 'Fiscal Position',
+            domain="[('fiscal_operation_id','=',fiscal_operation_id)]"),
```

Comment:
@rvalyi @renatonlima it seems that this domain comes from a portugal/brazil localization, I find it here: https://github.com/proge/openerp-pt_br/blob/master/l10n_br_stock/stock.py#L12
I simply duplicate the existing code for in & out pickings, but it seems that this domain could be removed.